src/components/cc-input-number/cc-input-number.stories.js

```diff
@@ -174,6 +174,35 @@ export const customWidth = makeStory(conf, {
 
 export const allFormControls = allFormControlsStory;
 
+export const simulation = makeStory(conf, {
+  items: [{}],
+  simulations: [
+    storyWait(0, ([component]) => {
+      component.innerHTML = `
+        <p slot="help">No error slot, no focus</p>
+      `;
+    }),
+    storyWait(2000, ([component]) => {
+      component.innerHTML = `
+        <p slot="help">With error, no focus</p>
+        <p slot="error">This is an error message</p>
+      `;
+    }),
+    storyWait(2000, ([component]) => {
+      component.innerHTML = `
+        <p slot="help">With error, with focus</p>
+        <p slot="error">This is an error message</p>
+      `;
+      component.focus();
+    }),
+    storyWait(2000, ([component]) => {
+      component.innerHTML = `
+        <p slot="help">No error slot, with focus</p>
+      `;
```

Comment:
Not really needed, as the component already has focus from the last step.
Maybe I could add it in case the user moved the focus between the step before and this one.

Reply:
Ho, actually, it works well in Firefox but the focus ring disappears in the last step when testing with Chrome. Can you confirm? :sweat_smile:

Reply:
Wow! good catch! I didn't try with Chrome and you're right.
So my last two commits will resolve the problem.